docs/services/slam/cartographer/_index.md

```diff
@@ -175,19 +189,71 @@ See [Attributes](#attributes) for details.
 }
 ```
 
-{{% /expand%}}
+{{% /tab %}}
+{{< /tabs >}}
+
+1. Start a mapping session
+
+   Navigate to the **Control** tab on your robot's page and click on the drop-down menu matching the `name` of the service you created.
+   On the cartographer panel, you can start a mapping session.
 
-#### Update existing map
+   When you start a mapping session:
 
-This mode is similar to [Create new map](#create-new-map), except it creates a new version of an existing map in the robot's location. (It does not overwrite the old version of the map.)
+   - Viam spins up an instance of the `cartographer` module in the cloud to execute the Cartographer algorithm.
+     Because Cartographer's algorithm is CPU-intensive when creating or updating a map, in this mode the `cartographer` module on your robot acts as a stub.
```

Comment:
I don't think we need to mention that Cartographer is CPU-intensive when creating or updating a map again here, since we already mentioned it in "Mapping Modes"